pkg/estimator/server/estimate.go

```diff
@@ -76,6 +76,12 @@ func (es *AccurateSchedulerEstimatorServer) estimateReplicas(
 
 	var res int32
 	replicas, ret := es.estimateFramework.RunEstimateReplicasPlugins(ctx, snapshot, &requirements)
+
+	// No replicas can be scheduled on the cluster, skip further checks and return 0
+	if ret.IsUnschedulable() {
+		return replicas, nil
```

Comment:
I'm afraid some readers may be confused about what the value of the `replicas` field is. can you return 0 explicitly here?